src/clear-all-refinements/index.ts

```diff
@@ -0,0 +1,11 @@
+import { tag, Tag } from '@storefront/core';
+
+@tag('gb-clear-all-refinements', require('./index.html'))
+class ClearAllRefinements {
+
+  onClick = () => this.actions.resetRefinements();
```

Comment:
I would make this an actual instance method 
```js
class MyClass {

  onClick() {
    this.actions.resetRefinements();
  }
}
```

I find it's more useful to make these arrow functions only when we need them to be (to preserve references to `this`) otherwise it becomes unclear why we would ever use one over the other